frontend/src/metabase/entities/users/forms.js

```diff
@@ -79,9 +82,23 @@ export default {
       ...PLUGIN_ADMIN_USER_FORM_FIELDS,
     ],
   },
-  user: {
-    fields: [...getNameFields(), getEmailField(), getLocaleField()],
-    disablePristineSubmit: true,
+  user: user => {
+    const isSsoUser = !PLUGIN_SHOW_CHANGE_PASSWORD_CONDITIONS.every(f =>
```

Comment:
We can reuse this logic to determine if a user logs in with SSO or not. See [Slack thread](https://metaboat.slack.com/archives/C03E8MQJZBM/p1655219016147239) here for more context.

Reply:
Yeah, the expression checking if a user is coming from SSO is a bit peculiar: it's not that clear how "password change conditions" are related to SSO (I assume there's no password for a user in this case) and `f` callback name is also not really clear. Wonder if we should maybe extract it to be a separate handler like `isSsoUser(user)`. It's worth leaving a comment explaining what's happening too I think

Reply:
What if we change the plugin name to `PLUGIN_IS_SSO_USERS` or `PLUGIN_IS_USER_PASSWORD_USERS`? Maybe it would tell the story better what do you think?

Reply:
And oh the `f` thing was deliberately copied from https://github.com/metabase/metabase/blob/11fdea21e793a10d83ed9c0b42f80d8a48973c5b/frontend/src/metabase/account/app/components/AccountHeader/AccountHeader.jsx#L21 just to aid searching in the future. I agree the name isn't ideal though 😅

Reply:
Apparently, this is the only change I applied to the PR.